### DISCUSSION TITLE
Can we run simulation forever

### DISCUSSION THREAD

**Discussion by SinghAS18** [CURENT/andes#387] [Q&A]
Hi everyone,

I am working with Andes for a quite sometime and we have successfully integrated it with our solution.

I'm looking, if it is possible to run the TDS (in qrt mode) forever. I mean I don't want to mention the time frame. I want it to keep running until user stops it.

Thanks,
SinghAS

**Comment by cuihantao** [comment]
You just need to disable outputting the result since running it for long will fill up your memory or disk quickly. See the simulation output control here: https://docs.andes.app/en/latest/release-notes.html#v1-7-0-2022-05-22
